radiant/missing.cpp

```diff
@@ -56,7 +56,9 @@ bool radCopyFile( const char *lpExistingFileName, const char *lpNewFileName, boo
 	char realsrc[PATH_MAX], realdest[PATH_MAX];
 
 	realpath( lpExistingFileName, realsrc );
-	realpath( lpNewFileName, realdest );
+#ifndef __OpenBSD__
+  realpath( lpNewFileName, realdest );
+#endif
```

Comment:
"realpath return empty string for an inexisting file" - doesn't sound like a BSD specific problem, so the fix should apply everywhere?

Reply:
is not all BSDs specific problem only for OpenBSD ^^.
for example on NetBSD i don't get it